### PR TITLE
supported exporters and status of each

### LINKED ISSUE #285: The list of supported backends for each exporter should be in `docs/`
Backends are listed in the various READMEs in `exporters/`, but we should have a canonical list available at readthedocs.

A rich table of what is supported will also be handy for setting expectations-- for instance, calling out #193 so new users don't have to find out the hard way.

### PR DESCRIPTION
**Pull request by weshayutin**
Issue: https://github.com/konveyor/pelorus/issues/285

## Linked Issues?

resolves #285 

related to #<issue number> <-- Use this if it shouldn't

## Testing Instructions

Please include any additional commands or pointers in addition to our [standard PR testing process](/docs/Development.md#testing-pull-requests).

@redhat-cop/mdt

### CODE REVIEW

File: docs/UpstreamSupport.md [hunk 1]
```diff
@@ -8,6 +8,39 @@ The Pelorus engineering team will provide `best-effort` level support for Peloru
 
 * You may also consider opening a [discussion](https://github.com/konveyor/pelorus/discussions) thread to discuss any feature, bug, or enhancement prior to opening a Github issue.
 
+## Backend Exporters
+
+Each of the three Pelorus exporters, the committime, deploytime and the failure exporter support various backends or services.  The Pelorus team is working to ensure an optimal experience for each backend service that is integrated with Pelorus.  Our goal is have each backend integrated into OpenShift's Prow Continuous Integration system to help ensure the quality of the integration of the service and Pelorus.
```
Comment (KevinMGranger) [inline docs/UpstreamSupport.md:13]: Might I suggest a more minimalist version:

> Pelorus's exporters support various backends or services. The Pelorus team is working to ensure an optimal experience for each backend integrated with Pelorus. Our goal is for each backend to be checked by OpenShift's Prow Continuous Integration system, to ensure the quality of each service's integration with Pelorus.

File: docs/UpstreamSupport.md [hunk 2]
```diff
@@ -8,6 +8,39 @@ The Pelorus engineering team will provide `best-effort` level support for Peloru
...
+
+Below is a list of the status of each backend per Pelorus exporter.
+
+### **Committime Exporter**
```
Comment (KevinMGranger) [inline docs/UpstreamSupport.md:17]: Do we have a formal definition for status / no known issues / known issues? Do we just mean major issues, or is this an assessment of an exporter's overall quality?

Comment (weshayutin) [reply docs/UpstreamSupport.md:17]: We have nothing formal, I was hoping to have some notion of a simple tier to help set expectations.
[ No known issues, known issues ]
e.g. service now I think we know a user would not have a decent experience.
Feel free to let me know if other troublesome data providers are there

[ In CI, not yet ]  Pretty easy to grok imho with the ultimate goal of having what ever possible to CI, in CI.

File: docs/UpstreamSupport.md (hunk 1 above)
Comment (weshayutin) [reply docs/UpstreamSupport.md]: k.. thanks for the simplification ++ added

File: docs/UpstreamSupport.md (hunk 2 above)
Comment (weshayutin) [reply docs/UpstreamSupport.md:17]: @mpryc please weigh in here

Comment (mpryc) [reply docs/UpstreamSupport.md:17]: To be honest I don't understand the table below. Is this something that will be updated every time new know issue is found? Or is this the expected community support (on the side note I would rename Upstream Support to Community Support).

We could say (added last sentence): 
"
## Backend Exporters

Pelorus's exporters support various backends or services. The Pelorus team is working to ensure an optimal experience for each backend integrated with Pelorus. Our goal is for each backend to be checked by OpenShift's Prow Continuous Integration system, to ensure the quality of each service's integration with Pelorus.

If new issue is discovered by Continues Integration system or filled manually, Pelorus team will triage it and prioritize respectively.
"

And than table may have links to appropriate labels within issues (we have some of them, but we could add per backend?).

Comment (KevinMGranger) [reply docs/UpstreamSupport.md:17]: If the idea was something like a "support tier", that makes sense.

Readiness for production:
Tier 1: ready for production. The pelorus team is confident in its quality.
Tier 2: mixed. The pelorus team is mostly confident in its quality, but is aware of some minor deficiencies.
Tier 3: experimental. The exporter may be used in production but users should expect some issues or rough corners.
Tier 4: will delete your computer and kick your dog.

Comment (weshayutin) [reply docs/UpstreamSupport.md:17]: @KevinMGranger I agree, IMHO the goal is to give some measure of confidence to folks using each backend.  How we phrase that could be Tier 1,2,3 etc..   We could also disambiguate the key... to be [ production, known issues, experimental ].  I like Tier, 1,2,3  I think once you hit experimental that's all we need.   @mpryc does this make sense to you?  If so let's decide on terms in the sync and move forward. :)

**Comment by mpryc** [comment]
@weshayutin this lgtm, however please consider my comment as your status link is different from what I meant, still good enough to serve the purpose.

**Comment by KevinMGranger** [comment]
What Migi said. I also thought we were doing this in addition to a "tier list"? Either way, this is a good start.

**Pull request merged**